addon/mixins/route.js

```diff
@@ -217,6 +216,19 @@ export default Ember.Mixin.create({
     return false;
   },
 
+  /**
+   Update the infinity model with new objects
+
+   @method updateInfinityModel
+   @param {Ember.Enumerable} newObjects The new objects to add to the model
+   @return {Ember.Array} returns the updated infinity model
+   */
+  updateInfinityModel(newObjects) {
+    var infinityModel = this.get(this.get('_modelPath'));
+
+    return infinityModel.pushObjects(newObjects);
+  },
+
```

Comment:
I think the example you gave wouldn't be possible if we did the `get('content')` inside this method (without an override).